### PR TITLE
Feature/collectd exporter

### PR DESCRIPTION
**Pull request by mindriot88**
#### Pull Request (PR) description
* Adds support for [collectd_exporter](https://github.com/prometheus/collectd_exporter)

### CODE REVIEW

File: manifests/collectd_exporter.pp [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+#  State ensured for the service (default 'running')
+#
+#  [*user*]
+#  User which runs the service
+#
+#  [*version*]
+#  The binary release version
+#
+class prometheus::collectd_exporter (
+  String $download_extension,
```
Comment (bastelfreak) [inline manifests/collectd_exporter.pp]: for all strings, could you please switch to `String[1]`? That will enforce a minimal string length of one.

**Comment by bastelfreak** [comment]
Thanks for the PR @mindriot88. Can you add an acceptance test to https://github.com/voxpupuli/puppet-prometheus/tree/master/spec/acceptance ?

**Comment by mindriot88** [comment]
@bastelfreak I have addressed both your comments, let me know if you need further changes

**Comment by bastelfreak** [comment]
I restarted one travis job. it failed because of a network issue between travis and github.

**Comment by bastelfreak** [comment]
Thanks!

**Pull request merged**